diagnostics/diagnostics/imb/Config/config_diags_ilamb.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0"?>
+
+<!-- =========================================================================== -->
+<!-- ILAMB Wrapper XML                                                           -->
+<!-- =========================================================================== -->
+
+<config_definition version="1.0">
+
+  <!-- =========================================================================== -->
+  <!-- ilamb_settings - general settings to be applied to all diagnostics routines   -->
+  <!-- NOTE: case specific environment variables from env_*.xml will be            -->
+  <!-- gathered at runtime by the ilamb_diags_generator.py python script.            -->
+  <!-- They do not need to be specified in this XML file.                          -->
+  <!-- The full $PATH variable will be set in the ilamb_diags_generator.py file      -->
+  <!-- using all the path settings defined here.                                   -->
+  <!-- =========================================================================== -->
+
+  <groups>
+    
+    <group name="global">
+      <order>0</order>
+      <comment>Environment variables to control ILAMB</comment>
+
+      <entry id="ILAMBDIAG_VERSION"
+             type="char"
+             valid_values=""
+             value="ilamb_version"
+             group="diags_ilamb"
+             desc="Package Version"
+             ></entry>
+
+    </group>
+    
+    <group name="ilamb_env">
+      <order>1</order>
+      <comment>ILAMB Environment Variables</comment>
+
+      <entry id="ILAMBDIAG_MPLBACKEND"
+             type="char"
+             valid_values="Agg"
+             value="Agg"
+             group="ilamb_env"
+             desc="matplotlib backend for generating graphics, should be exported to the environment!"
+             ></entry>
+
+      <entry id="ILAMBDIAG_ILAMB_ROOT"
+             type="char"
+             valid_values="Agg"
+             value="/glade/p/work/aliceb/sandboxes/tests/ILAMB"
+             group="ilamb_env"
+             desc="environment variable model data root path"
+             ></entry>
```

Comment:
Please change these entries to be:
valid_values="/path/to/ILAMB/data"
value="/glade/p/cesm/lmwg/ilamb/ILAMB_ROOT"
desc="environment variable model data root path. These are data files downloaded using the ilamb-fetch command."

Reply:
updated in latest version.